app/views/pages/data_changelog.html.haml

```diff
@@ -21,9 +21,28 @@
             as a user of our data. Where possible, we'll identify directly
             which releases of bulk data you'll see these changes in.
 
-          %h2 July 2023
+          %h2 9 September 2023
           %p
-            <strong>First affected bulk download:</strong> July 2023
+            <strong>First affected bulk download:</strong> 9 September 2023
+          %ul
+            %li
+              %p
+                Legal Entity Identifiers (LEIs) have been added as additional 
+                identifiers to enabling greater connectivity between international 
+                beneficial ownership  and corporate ownership datasets.
+              %p
+                This <a href="https://github.com/openownership/register-sources-oc/issues/8">work</a> 
+                has been enabled by an <a href="https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files">existing 
+                mapping process</a> between OpenCorporates IDs and LEIs.
+              %p
+                Technical details about mapping Open Ownership Register IDs to LEIs has been 
+                <a href="https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files/open-ownership-register-id-to-lei-relationship">
```

Comment:
This style of HTML-within-HAML will work, but a HAML-only alternative would be something like

```haml
%a{ href: 'https://www.gleif.org/en/lei-data/lei-mapping/download-oc-to-lei-relationship-files/open-ownership-register-id-to-lei-relationship' } published
```

etc.